fix_includes.py

```diff
@@ -2373,8 +2373,7 @@ def main(argv):
   parser.add_option('--nocomments', action='store_false', dest='comments')
 
   parser.add_option('--update_comments', action='store_true', default=False,
-                    help=('Update #include comments, even if no #include lines'
-                          ' are added or removed'))
+                    help=('replace \'why\' comments with the ones provided by IWYU'))
```

Comment:
This runs over 80 column limit, needs to be wrapped.
All other help texts use sentence case, so this should probably start with an upper-case R.